### PR TITLE
Update the invenio and pride packages

### PR DESCRIPTION
**Pull request by LibraChris**
- Ensure that the invenio and pride packages are upgraded to align with ARCTokenisation version 5.0.0.
- Implement minor adjustments to enhance code clarity and bolster test robustness.

### CODE REVIEW

**Comment by LibraChris** [comment]
Hey @omaus,@kMutagene ,
could you have a look at this and review it please?

**Comment by kMutagene** [comment]
We should definitely use this iccasion to also get in the new ARCExpect API and binding frontmatter to prevent a new package release immediately after this one.

More on that next week after i have properly documented and released https://github.com/nfdi4plants/arc-validate/pull/101

**Comment by omaus** [comment]
> Hey @omaus,@kMutagene , could you have a look at this and review it please?

Yes, I'll have a look into it next week. :)

File: StagingArea/pride/pride@1.0.0.fsx [hunk 1]
```diff
@@ -34,174 +38,77 @@ ReleaseNotes: |
 ---
 *)
 
+#r "nuget: ARCTokenization,5.0.0"
 #r "nuget: ARCExpect"
 #r "nuget: Anybadge.NET"
-#r "nuget: ARCValidationPackages"
 #r "nuget: FSharpAux"
```
Comment (omaus) [inline StagingArea/pride/pride@1.0.0.fsx:44]: Do you even need this anymore?

**Review by omaus — COMMENTED** [review]
In general, lgtm.

**Comment by kMutagene** [comment]
@omaus @LibraChris i adapted the packages to the latest possibilities. Please check on your machines whether these packages still create the expected output

**Comment by kMutagene** [comment]
reminder that this PR switches to _running_ scripts instead of compiling them for sanity checks. This is obviously not a situation we want to stay in, so fingers cross we can get a resolution on https://github.com/dotnet/fsharp/issues/17159

**Review by omaus — APPROVED** [review]
Just a small question but otherwise lgtm.

**Pull request merged**